cristin-commons/src/main/java/no/unit/nva/utils/AccessUtils.java

```diff
@@ -54,10 +50,7 @@ public class AccessUtils {
      * @throws ForbiddenException thrown when user is not authorized to access a user with IdentificationNumber
      */
     public static void validateIdentificationNumberAccess(RequestInfo requestInfo) throws ForbiddenException {
-        if (requesterHasNoAccessRightToUseNationalIdentificationNumber(requestInfo)) {
-            String nvaUsername = attempt(requestInfo::getUserName).orElse(fail -> null);
-            logger.warn(USER_DOES_NOT_HAVE_REQUIRED_ACCESS_RIGHT,
-                    nvaUsername, MANAGE_OWN_AFFILIATION);
+        if (!requesterIsUserAdministrator(requestInfo)) {
```

Comment:
if one checks this method `requesterIsUserAdministrator `one can see that it is equal to the one removed, apart from the negation which i have added here

Reply:
Kept the logging